docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.8"
+
+x-app-common:
+  &app-common
+  image: event_app:latest
+  build: .
+  env_file:
+    - .env.${APP_ENV}
+  networks:
+    - db
+    - cache
+
+services:
+  app:
+    <<: *app-common
+    command: >
+      gunicorn event_app.wsgi:application
+        --name app-gunicorn
+        --bind 0.0.0.0:8000
+        --workers 4
```

Comment:
its better that these configs like worker count come from an env file

Reply:
also port

Reply:
The worker count here isn't really that important, because the scaling should be done by docker-compose (or Kubernetes). Worker count is set to 4 because it's the optimal number for each instance of Gunicorn.

The port is also the local one, the public port number is present in the .env file